internal/generated/openapi/zz_generated.openapi.go

```diff
@@ -1,7 +1,7 @@
 //go:build !ignore_autogenerated
 // +build !ignore_autogenerated
 
-// Copyright 2022 New Relic Corporation. All rights reserved.
+// Copyright 2023 New Relic Corporation. All rights reserved.
```

Comment:
This change is the result of running `make generate`, as required by the CI checks:
https://github.com/newrelic/newrelic-k8s-metrics-adapter/blob/main/Makefile#L65